packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -3574,6 +3574,27 @@ describe('scope hoisting', function () {
   });
 
   describe('commonjs', function () {
+    it('should wrap when this could refer to an export', async function () {
+      let b = await bundle(
+        path.join(__dirname, '/integration/exports-this/a.js'),
+        {
+          mode: 'production',
+          defaultTargetOptions: {
+            shouldScopeHoist: true,
+            shouldOptimize: false,
+          },
+        },
+      );
+
+      let contents = await outputFS.readFile(
+        b.getBundles().find(b => /a\.js/.test(b.filePath)).filePath,
+        'utf8',
+      );
```

Comment:
These custom build options are already set at the top of this file, and there is exactly one bundle in the test case, no need to use a regex.
```suggestion
      let b = await bundle(
        path.join(__dirname, '/integration/exports-this/a.js'),
      );

      let contents = await outputFS.readFile(
        b.getBundles()[0].filePath,
        'utf8',
      );
```